### DISCUSSION TITLE
Player Teams

### DISCUSSION THREAD

**Discussion by Akornzzz** [ppy/osu#25652] [Ideas] [answered]
You know how a lot of games have 'Clans'? Well, I think that adding something similar could be a fun little feature. 

The Idea is something like this;
-Create a Unique Team Name (you could charge for creating a team because it is just cosmetic)
-Join a Team (only if that team is public or the leader invites you)
-Compete as a team to get to the top (some sort of ranked team duels would be cool too)

This is a very loose idea so take it however you please.

**Comment by huyenden** [comment]
I can imagine the chats causing war between guilds and solo players feeling bothered about it 🤣

**Comment by peppy** [comment, marked as answer]
Is already in progress.
